external-docs/docs/guides/configuration/qube-config/columns/observations.md

```diff
@@ -0,0 +1,92 @@
+# Observations columns
+
+This page discusses what an observations column is, where one should be used, and how one can be defined.
+
+> For a detailed look at an observations column's configuration options, see the [Reference table](#reference) at the bottom of this page.
+
+## What is an observations column?
+
+Observations columns contain the numerical values of observations recorded in the data set, and are the most important component of a cube. In order to be valid, a data cube must include at least one observations column, each of which must have a unit and a measure associated with it. Measures and units can either be defined against the observations column, or can be contained in separate [units](./units.md) and [measures](./measures.md) columns.
+
+## When to use one
+
+Observations columns contain the observed values of your data, and as such your data set must always contain at least one observations column. The configuration of observations columns in your data set will primarily depend on the [shape of your data](../../../shape-data/index.md). This is discussed in more detail below.
```

Comment:
You've used observations columns a bit too much in this paragraph. Can you get rid of at lease one?